### PR TITLE
core and non core

### PR DESCRIPTION
**Pull request by WHYTEWYLL**

### CODE REVIEW

File: .github/workflows/dbt_run_core_scheduled.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ run-name: dbt_run_scheduled
 on:
   workflow_dispatch:
   schedule:
-    # Runs "every hour" (see https://crontab.guru)
-    - cron: "30 * * * *"
+    # Runs "every 20mins" (see https://crontab.guru)
+    - cron: "20 * * * *"
```
Comment (eric-laurello) [inline .github/workflows/dbt_run_core_scheduled.yml:8]: I would change this to "20,40 * * * *". We can run core and noncore combined in the hourly job. We want to do this so all the models run in the right order.

File: .github/workflows/dbt_run_non_core_scheduled.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          cache: "pip"
+
+      - name: install dependencies
+        run: |
+          pip install -r requirements.txt
+          dbt deps
+
+      - name: Run DBT Jobs
+        run: |
+          dbt seed; dbt run -s tag:scheduled_non_core+ --exclude streamline__s3_sync tag:livequery tag:atlas tag:exclude_from_schedule
```
Comment (eric-laurello) [inline .github/workflows/dbt_run_non_core_scheduled.yml:45]: should just need to add "tag:scheduled_core" to run core & non core within this

File: models/silver/actions/silver__actions_events_function_call_s3.sql [hunk 3]
```diff
@@ -4,7 +4,7 @@
   merge_exclude_columns = ["inserted_timestamp"],
   cluster_by = ['block_timestamp::DATE', '_inserted_timestamp::DATE'],
   unique_key = 'action_id',
-  tags = ['actions', 'curated']
+  tags = ['actions', 'curated','scheduled_non_core']
```
Comment (eric-laurello) [inline models/silver/actions/silver__actions_events_function_call_s3.sql:7]: Probably should be core if this is a dependency of core.fact_actions_events_function_call

File: models/silver/actions/silver__actions_events_s3.sql [hunk 4]
```diff
@@ -4,7 +4,7 @@
     merge_exclude_columns = ["inserted_timestamp"],
     cluster_by = ['block_timestamp::DATE', '_inserted_timestamp::DATE'],
     unique_key = 'action_id',
-    tags = ['actions', 'curated']
+    tags = ['actions', 'curated','scheduled_non_core']
```
Comment (eric-laurello) [inline models/silver/actions/silver__actions_events_s3.sql:7]: same here with core. Should double check all the Core dependencies.

File: .github/workflows/dbt_run_core_scheduled.yml [hunk 5]
```diff
@@ -42,8 +42,8 @@ jobs:
 
       - name: Run DBT Jobs
         run: |
-          dbt run-operation dispatch_github_workflow --args "{'repo_name': 'streamline-snowflake', 'workflow_name': 'dbt_run_near_external_table_update', 'gb_id': '${{ secrets.GB_ID}}'}";
-          dbt seed; dbt run -s models/bronze tag:helper tag:load+ tag:labels+ tag:scheduled_non_core --exclude streamline__s3_sync tag:livequery tag:atlas tag:exclude_from_schedule
+          dbt run-operation dispatch_github_workflow --args "{'repo_name': 'streamline-snowflake', 'workflow_name': 'dbt_run_near_external_table_update', 'gb_id': '${{ secrets.GB_ID}}', 'RECEIPT_MAP_LOOKBACK_HOURS': 0.2}}";
```
Comment (forgxyz) [inline .github/workflows/dbt_run_core_scheduled.yml:45]: 0.2 hrs is 12 minutes which won't be sufficient. Let's keep it 0.5 hrs for the first set of runs and monitor.

File: .github/workflows/dbt_run_non_core_scheduled.yml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          cache: "pip"
+
+      - name: install dependencies
+        run: |
+          pip install -r requirements.txt
+          dbt deps
+
+      - name: Run DBT Jobs
+        run: |
+          dbt seed; dbt run -s tag:scheduled_non_core+
```
Comment (forgxyz) [inline .github/workflows/dbt_run_non_core_scheduled.yml:45]: I don't know what `dbt run-operation dispatch_github_workflow` in the core version is, but why do we only need it there and not here?

File: models/silver/curated/silver__logs_s3.sql [hunk 7]
```diff
@@ -4,7 +4,7 @@
     cluster_by = ["_inserted_timestamp::DATE","block_timestamp::DATE"],
     unique_key = "log_id",
     incremental_strategy = "merge",
-    tags = ['curated']
+    tags = ['curated','scheduled_non_core']
```
Comment (forgxyz) [inline models/silver/curated/silver__logs_s3.sql:7]: I can see an argument for logs being core

File: models/silver/curated/defi/bridge/silver__bridge_multichain.sql [hunk 8]
```diff
@@ -4,7 +4,7 @@
     merge_exclude_columns = ["inserted_timestamp"],
     unique_key = 'bridge_multichain_id',
     cluster_by = ['block_timestamp::DATE', '_modified_timestamp::DATE'],
-    tags = ['curated', 'exclude_from_schedule'],
+    tags = ['curated', 'exclude_from_schedule','scheduled_non_core'],
```
Comment (forgxyz) [inline models/silver/curated/defi/bridge/silver__bridge_multichain.sql:7]: Multichain bridge was hacked and is inactive. Can leave untagged so it doesn't run and waste compute doing scans

**Review by forgxyz — APPROVED** [review]

**Comment by forgxyz** [comment]
LGTM

**Comment by forgxyz** [comment]
TODO on merge - update GitHub Warehouse Var to `dbt_cloud_large`

**Pull request merged**